.azure-pipelines/deploy-permissions-scraper.yml

```diff
@@ -0,0 +1,76 @@
+# .NET Core Function App to Windows on Azure
+# Build a .NET Core function app and deploy it to Azure as a Windows function App.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/dotnet-core
+
```

Comment:
Let's add a verbose naming convention that helps when having to run this multiple times.

`name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)`